pkg/identityManager/identityManager_test.go

```diff
@@ -90,8 +85,10 @@ var _ = Describe("IdentityManager", func() {
 		restConfig = cluster.GetCfg()
 
 		namespaceManager = tenantnamespace.NewTenantNamespaceManager(client)
-		identityMan = NewCertificateIdentityManager(cluster.GetClient(), localClusterID, namespaceManager)
-		identityProvider = NewCertificateIdentityProvider(ctx, cluster.GetClient(), localClusterID, namespaceManager)
+		identityMan = NewCertificateIdentityManager(cluster.GetClient(),
+			localClusterID, namespaceManager)
+		identityProvider = NewCertificateIdentityProvider(ctx, cluster.GetClient(),
+			localClusterID, namespaceManager)
```

Comment:
Are those changes intentional? Would personally avoid too short lines, but that is definitely opinionated. (Same for the occurrences below)